downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -1273,6 +1273,16 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
 
 - To avoid clashes with Java identifiers, the character used for encoding and decoding identifiers has been changed from `$` to `&`.
 
+- A bug which resulted in no compilation errors for interop calls of Java abstract class constructors has been fixed.
```

Comment:
```suggestion
- A bug that allowed interop calls of Java abstract class constructors to proceed without generating compilation errors has been fixed.
```
This sounds a bit more explanatory. Is it too long?

Reply:
Stick with the original sentence to follow the writing style of this doc.